app/views/requests/edit.html.erb

```diff
@@ -8,6 +8,19 @@
       <h1 class="govuk-heading-xl"><%= content_for?(:title) ? yield(:title) : t("request_form.#{@form.name}") %></h1>
       <%= yield :form_content %>
       <%= f.govuk_submit %>
-    <% end %>
+      <% if content_for?(:text) || t("request_form.#{@form.name}.self_details", default: '').present? %>
+        <details class="govuk-details">
```

Comment:
Please use the component - https://govuk-components.netlify.app/components/details/#output-html-details-with-a-block

Reply:
Is there a reason not to update this?